setup.py

```diff
@@ -33,11 +33,12 @@
         "Programming Language :: Python :: 3.10",
         "Programming Language :: Python :: 3.11",
         "Programming Language :: Python :: 3.12",
+        "Programming Language :: Python :: 3.13",
     ],
     description="The RelationalAI Software Development Kit for Python",
     install_requires=[
         "pandas>=2.0.0,<3.0.0;python_version>'3.8'",
-        "pyarrow>=10.0.0,<16.0.0",
+        "pyarrow==22.0.0",
```

Comment:
I don't think we can pin to exactly version 22. It requires Python 3.10 https://github.com/apache/arrow/blob/main/python/setup.py#L29, but relationalai-python says it supports 3.9 too https://github.com/RelationalAI/relationalai-python/blob/main/README.md?plain=1#L57.

How about a more conservative change `pyarrow>=10.0.0,<23.0.0`